standard/grammar.md

```diff
@@ -10,15 +10,15 @@ This annex contains the grammar productions found in the specification, includin
 
 ```ANTLR
 
-// Source: §6.3.1 General
+// Source: [§6.3.1](lexical-structure.md#631-general) General
```

Comment:
Why are these links now here? I can't see a code change that would cause them... and I don't think we should have them, given that they don't render as links anyway.